docs/pages/customization/unstyled-components.js

```diff
@@ -0,0 +1,11 @@
+import * as React from 'react';
+import MarkdownDocs from 'docs/src/modules/components/MarkdownDocs';
+import {
+  demos,
+  docs,
+  requireDemo,
+} from '!@material-ui/markdown/loader!docs/src/pages/customization/unstyled-components/unstyled-components.md';
```

Comment:
@eps1lon changed this

```suggestion
} from 'docs/src/pages/customization/unstyled-components/unstyled-components.md?@material-ui/markdown';
```